dired-narrow.el

```diff
@@ -241,7 +241,14 @@ read from minibuffer."
         (progn
           (dired-narrow-mode 1)
           (add-to-invisibility-spec :dired-narrow)
-          (setq disable-narrow (read-from-minibuffer "Filter: " nil dired-narrow-map))
+          (setq disable-narrow (read-from-minibuffer
+                                (pcase dired-narrow-filter-function
+                                  ('dired-narrow--regexp-filter
+                                   "Regex Filter:\s")
```

Comment:
What are the `\s` for?

Reply:
Simply to represent a space character; it's a habit I use when programming to reduce accidental adding or removal of spaces.